api/v3/common_types.go

```diff
@@ -138,6 +145,8 @@ type StorageClassSpec struct {
 	StorageCapacity string `json:"storageCapacity"`
 
 	// If true, ephemeral (emptyDir) storage will be used
+	// default false
+	// +optional
```

Comment:
it's new for me. does this notation work for the marker? similarly to other places

Reply:
its been used in k8s controller code